pkg/reconciler.v1/common/utils.go

```diff
@@ -22,9 +22,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const ReconcilerName = "Kubeflow Reconciler"
+const ReconcilerName = "common-reconciler"
 
-// GetReconcilerName returns the name of this reconciler, which is "Kubeflow Reconciler"
+// GetReconcilerName returns the name of this reconciler, which is "common-reconciler"
 func (r *ReconcilerUtil) GetReconcilerName() string {
```

Comment:
"Kubeflow Reconciler" won't work because it's not a word?

Reply:
Yes. When generating the default labels, the returned string from `GetReconcilerName` will be used as the value for key `commonv1.OperatorNameLable` (`""training.kubeflow.org/operator-name""`).